frontend/public/actions/ui.ts

```diff
@@ -178,14 +178,14 @@ export const stopImpersonate = () => (dispatch) => {
   dispatch(detectFeatures());
   history.push(window.SERVER_FLAGS.basePath);
 };
-export const sortList = (listId: string, field: string, func: any, orderBy: string, column: string) => {
+export const sortList = (listId: string, field: string, func: any, sortAsNumber: boolean, orderBy: string, column: string) => {
```

Comment:
Not new, but shouldn't `func` be `string` here?

This is actually another reason why we shouldn't pass a function as before. The value is stored in redux, and functions aren't serializable.

https://stackoverflow.com/questions/35325195/should-i-store-function-references-in-redux-store

Reply:
fixed